testdata/baselines/reference/submodule/conformance/privateNameMethodClassExpression.errors.txt.diff

```diff
@@ -13,10 +13,10 @@
      C.getInstance().#method; // Error
                      ~~~~~~~
 -!!! error TS18013: Property '#method' is not accessible outside class '(anonymous)' because it has a private identifier.
-+!!! error TS18013: Property '#method' is not accessible outside class '(Missing)' because it has a private identifier.
++!!! error TS18013: Property '#method' is not accessible outside class 'C' because it has a private identifier.
      C.getInstance().#field; // Error
                      ~~~~~~
 -!!! error TS18013: Property '#field' is not accessible outside class '(anonymous)' because it has a private identifier.
-+!!! error TS18013: Property '#field' is not accessible outside class '(Missing)' because it has a private identifier.
++!!! error TS18013: Property '#field' is not accessible outside class 'C' because it has a private identifier.
```

Comment:
I think it would be good to add this to accepted baselines but I'm not sure how to do it.

Reply:
Add this line to submoduleAccepted.txt:

```
privateNameMethodClassExpression.errors.txt.diff
```

Reply:
done